app/usecase/notification/listener_test.go

```diff
@@ -0,0 +1,66 @@
+package notification
+
+import (
+	"testing"
+	"time"
+
+	"github.com/asaskevich/EventBus"
```

Comment:
Application shouldn't depend on third party libraries. EventBus should be abstracted out through interfaces. For testing, we should use the fake implementation provided by mdtest package instead.

Reply:
You are right!

It means, I need to add a new interface `EventBus` to `app` repository, and then we can solve this issue. If [this](https://github.com/short-d/kgs/pull/54/files#diff-7ddfb3e035b42cd70649cc33393fe32cR65) doesn't work, I will rewrite `app.EventDispatcher`